.travis.yml

```diff
@@ -25,24 +25,22 @@ matrix:
     env: TASK=test
   - node_js: 5
     env: TASK=test
-  - node_js: node
-    env: TASK=test
-  - node_js: node
-    env: TASK=browser BROWSER_NAME=opera BROWSER_VERSION="11..latest"
-  - node_js: node
+  - node_js: 5
+    env: TASK=browser BROWSER_NAME=android BROWSER_VERSION="4.0..latest"
+  - node_js: 5
     env: TASK=browser BROWSER_NAME=ie BROWSER_VERSION="9..latest"
-  - node_js: node
-    env: TASK=browser BROWSER_NAME=chrome BROWSER_VERSION="41..beta"
-  - node_js: node
-    env: TASK=browser BROWSER_NAME=firefox BROWSER_VERSION="36..latest"
-  - node_js: node
-    env: TASK=browser BROWSER_NAME=ipad BROWSER_VERSION="['6.1', '7.1', '8.2']"
-  - node_js: node
-    env: TASK=browser BROWSER_NAME=iphone BROWSER_VERSION="['6.1', '7.1', '8.2']"
-  - node_js: node
+  - node_js: 5
+    env: TASK=browser BROWSER_NAME=opera BROWSER_VERSION="11..latest"
+  - node_js: 5
+    env: TASK=browser BROWSER_NAME=chrome BROWSER_VERSION="-3..latest"
```

Comment:
sauce labs was failing on dev (failing to start the browser, tests where not gotten to) so switched to latest, also switched it to do a rolling 3 latest for this and firefox